投稿画面の実装

shizuka0727 opened the pull request:
投稿画面のHTML, SCCを作成しました。

shizuka0727 committed:
投稿画面の実装

投稿画面のHTML, SCCを作成しました。

shizuka0727 commented:
Whyを記載しておりませんでした。
chat-spaceのビューを完成させてから機能の作成に進むため

shizuka0727 commented:
すみません、キャプチャした画像を添付しておりませんでした。再送いたします。

TOMARUMARU commented on app/assets/stylesheets/_massages.scss:
```diff
@@ -0,0 +1,135 @@
+    width: 300px;
+    color: #FFFFFF;
+    }
+.chat__side--header {
```
ネストして＆を使いましょう！
他の箇所も同様です

shizuka0727 replied:
こちら、ネストして&を使用すると表示されなくなってしまいます。
色々なサイトを調べたのですが、原因が分かりません。
ありがちなミスなどございましたら、ご教授いただきたいです。。。

shizuka0727 replied:
ネストして記載することができるようになりました。
括り方が間違っていました。

TOMARUMARU commented on app/views/massages/index.html.haml:
```diff
@@ -0,0 +1,74 @@
+!!!
+%html
+%body
```
ここから上の記述は不要です

shizuka0727 replied:
削除いたしました。ありがとうございます。

TOMARUMARU commented on app/views/massages/index.html.haml:
```diff
@@ -0,0 +1,74 @@
+!!!
+%html
+%body
+  %div.flat__box--top グループを作成しました
```
divは省略できます

shizuka0727 replied:
divは省略して表記に修正いたしました。

TOMARUMARU commented on app/views/massages/index.html.haml:
```diff
@@ -0,0 +1,74 @@
+      <input type="text" class="posts__text-field">
+      <button type="submit" class="posts__submit">Send</button>
+      </form>
+      </div>
```
hamlで書きましょう

shizuka0727 replied:
hamlで記載して修正しました。有難うございます。

shizuka0727 committed:
投稿画面の実装　修正

shizuka0727 commented:
https://gyazo.com/3ae0f52fc9319c72e1b295a3c169a4c4

キャプチャ画面を送ります。
html.hamlファイルをブラウザで直接表示できなかったため、ルーティングの設定をして表示させています。
問題が、アイコンにリンクを設定すると、ムラサキ色になってしまい、csccファイルの方で、
colore: none, !important;
colore; #FFFFFF !important;
など記載しても修正できておりません。

shizuka0727 committed:
チャット画面の実装

・html.hamlファイルを画面に表示できず、ルーティングを設定して表示しております。
・画面左のアイコンのリンクは、アプリケーション内の他のページとしてそれぞれ設定
・については、formタグの使用のみ

shingoteshima commented on app/views/layouts/mailer.html.haml:
```diff
@@ -0,0 +1,8 @@
+!!!
```
mailerは今回は使用しないのでmailer系のファイルはすべて削除しましょう。

shizuka0727 replied:
確かにそうですね。有難うございます。削除いたしました。

shizuka0727 committed:
mailer系のフォルダの削除

miyagit commented on app/assets/stylesheets/_messages.scss:
```diff
@@ -0,0 +1,143 @@
+  &__side--new-groups {
+    padding-left: 4px;
+    }
+}
```
こちらインデントおかしくなってますね。修正しましょう。

shizuka0727 replied:
特に表示に問題ないかと思います。
一応、修正して確認いたしました。

shizuka0727 replied:
画面実装の次のカリキュラムに進みたいのですが、いかがでしょうか。

ken-nag replied:
表示には問題はなくても、コードの質を守るために修正お願いします

ken-nag commented on app/controllers/messages_controller.rb:
```diff
@@ -0,0 +1,12 @@
+  def index
+  end
+
+  def edit_user
```
user情報を編集するアクションはmessagesに置くべきではないですね

shizuka0727 replied:
インデント、controllerについて、修正いたしました！有難うございます！

ken-nag commented on app/controllers/messages_controller.rb:
```diff
@@ -0,0 +1,12 @@
+  def edit_user
+  end
+
+  def new_groups
```
groupの新規作成のアクションはmessagesに置くべきではないです

shizuka0727 replied:
そうですね、修正しました！

ken-nag commented on config/routes.rb:
```diff
@@ -1,3 +1,5 @@
-end
+  get 'messages' => 'messages#index'
+  get 'messages' => 'messages#edit_user'
+  get 'messages' => 'messages#new_groups'
```
上記でのmessages_controllerでコメントしたことからこちらの方も修正お願いします

shizuka0727 replied:
こちらも修正いたしました！

shizuka0727 committed:
修正版の提出

ken-nag commented:
lgtM
